### PR TITLE
Add formatting section

### PR DESCRIPTION
**Pull request by alexbirkett**

### CODE REVIEW

**Comment by Ehyeh-Asher-Ehyeh** [comment]
The long method declaration and chained method calls examples are both less than 80 chars long. Suggest making them longer to avoid confusion.

All the examples bare chained method calls do // bad first, then // good. Suggest having chained method also do this order for consistency.

File: java.md [hunk 1]
```diff
@@ -70,3 +70,146 @@ CammelCase must be used for emum names. Constants must be defined in SHOUTY_CASE
...
+}
+```
+
+### Braces in return statements
+
+Braces must not be used in return statements.
+
+```java
+// Bad
+return (x > y);
```
Comment (havchr) [inline java.md]: why bad? Sometimes you need braces in a return statement.

Comment (hfossli) [inline java.md]: When?

Comment (alexbirkett) [inline java.md]: You're right. You might need braces in return statements e.g. 
```
private static int calculateTwo(int x, int y, int z) {
        return (x + y) * z;
    }
```

I'll update the document to say that the braces should not wrap the entire return statement expression.

**Comment by alexbirkett** [comment]
Pull request updated @Ehyeh-Asher-Ehyeh and @havchr

**Pull request merged**